### DISCUSSION TITLE
How to change the effective date of a transaction

### DISCUSSION THREAD

**Discussion by KilianRosini** [moneymanagerex/moneymanagerex#6133] [Q&A]
Hi everyone, 

I just downloaded money manager ex, and I'm a huge fan! Although, I'm wondering how or if I can change the effective date of a transaction. For example: I get paid at the end of each month (for example – 28th of July) But I don't want it to be added to the month of July but rather for the month of august. 
Is there a way to change the effective date?

I would be really happy to hear from you!
Thanks in advance.

**Comment by vomikan** [comment]
Possibly the follow-up status designed for that.

**Comment by tactilis** [comment]
>  I'm wondering how or if I can change the effective date of a transaction. For example: I get paid at the end of each month (for example – 28th of July) But I don't want it to be added to the month of July but rather for the month of august.
Is there a way to change the effective date?

A MMEX account is intended to reflect the deposits to and the withdrawals from your real world bank account.

You got paid on 28th July. That's when your bank account balance increased, so that is what should be shown in MMEX.

However, if you _really_ want the MMEX account to show your pay being deposited on 1st Aug, then you will need to set the transaction date to 1st Aug. I'd suggest that you add a Note to say that the deposit was actually made on 28 Aug.

Note that doing this means that you won't be able to reconcile transactions in MMEX with your bank statement because the balances won't match on 28, 29, 30, 31 July.

**Comment by rmelillo76** [comment]
Welcome to the community Kilian! Martin was spot on that MMEX is designed to reflect the actual transactions posted to your bank account, and although you didn't ask specifically about budgeting, I'll provide a budgeting tip that could help, assuming you wanted to see how far your money will last into the new month.

I will first caveat that as a new user to MMEX, I would suggest you get familiar with some of the basics before delving into some of the more advanced options, but here goes:

From Tools -> Options -> Select View Options, then change **Budget offset (days)** to either -3 or -4. You can adjust this as needed depending on if you are always paid on the 28th and how many days are in the current month. The application will use this value to subtract the # of days specified from the 1st of each new month when building and showing your budget.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/95182117/4152c5e0-61a2-4aeb-a1ab-a7e71a758d9f)

That will cause the Budget (after you have created it) to start on the 27th or 28th, again depending on how many days there are in the month, with February of course being shorter.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/95182117/23326b76-fc07-4075-a0c6-7c6d8ce10583)

This transaction will now be shown under August's Budget view and reports.

![image](https://github.com/moneymanagerex/moneymanagerex/assets/95182117/8170a8e1-c3fb-431e-919c-5d6938250193)

(Note: I just spotted a defect where the Start Date is missing in this view , but does reflect for the entire year so I will raise that with the development team)
![image](https://github.com/moneymanagerex/moneymanagerex/assets/95182117/b6219d2a-b693-47d5-89aa-bbb26184c48c)

Again, this may overcomplicate things for what you're looking to do but I would encourage you to try things out and I hope you found this helpful!

Rich
